### DISCUSSION TITLE
Unable to use Moto G 5G 2023 to connect to Canon printer

### DISCUSSION THREAD

**Discussion by BJAdams** [community/community#125500] [Apps, API and Webhooks]
Remote control of Canon printer### Select Topic Area

Question How to connect Moto G phone to Canon printer

### Body

My smartphone is an optional remote device server for my Canon model TS6400 series printer. The device was able to print a creative document originally then failed upon further attempts. Correction required connecting to GitHub for resolution. Very frustrating and puzzling as a Google Pixel 7a can connect and print docs.

**Comment by queenofcorgis** [comment]
Hi @BJAdams ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Canon printers. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
